src/region/__tests__/RegionCreator-test.tsx

```diff
@@ -61,6 +62,16 @@ describe('RegionCreator', () => {
             act(() => {
                 document.dispatchEvent(new MouseEvent('mouseup'));
             });
+        const simulateMouseOver = (wrapper: ReactWrapper): void => {
```

Comment:
These are pretty simple and can be inlined in the test since they're only used once.